mmv1/products/dialogflowcx/api.yaml

```diff
@@ -560,3 +560,77 @@ objects:
           - :RUNNING
           - :SUCCEEDED
           - :FAILED
+  - !ruby/object:Api::Resource
+    name: 'Environment'
+    base_url: "{{parent}}/environments/"
+    self_link: "{{name}}"
```

Comment:
This self_link doesn't match with the import id, is the `name` the short name or the long name that includes the parent & environment?

Reply:
yes I removed self_link